window/runner_panel.py

```diff
@@ -29,7 +34,7 @@ def __init__(self, todds_dry_run_support=False):
         logger.info("Initializing RunnerPanel")
         self.ansi_escape = compile(r"\x1B\[[0-?]*[ -/]*[@-~]")
         self.system = system()
-
+        self.warningmod = []
```

Comment:
Let's call this `self.steamcmd_failed_mods` or something like that. Please also add a comment describing what it is being used for.